src/gateway/handlers/presenceUpdate.ts

```diff
@@ -0,0 +1,6 @@
+import { Gateway, GatewayEventHandler } from '../index.ts'
+
+export const presenceUpdate: GatewayEventHandler = async (
+  gateway: Gateway,
+  d: any
+) => {}

```

Comment:
what is it?

Reply:
Empty event for now, was going to work on it later.

Reply:
Ok